### PR TITLE
Add exclude regexes to triage

### PR DESCRIPTION
**Pull request by spiffxp**
I wanted to find tests that failed with a specific failure text, but didn't have 'sig-storage' in their name

eg: https://storage.googleapis.com/k8s-gubernator/triage/staging/index.html?text=OCI%20runtime%20start%20failed%3A%20container%20process%20is%20already%20dead%3A%20unknown&job=gce

### CODE REVIEW

File: triage/interactive.js [hunk 1]
```diff
@@ -50,9 +50,12 @@ function readOptions() {
...
     pr: read('job-pr'),
-    reText: read('filter-text'),
-    reJob: read('filter-job'),
-    reTest: read('filter-test'),
+    reText: read('filter-include-text'),
+    reJob: read('filter-include-job'),
+    reTest: read('filter-include-test'),
+    reXText: read('filter-exclude-text'),
+    reXJob: read('filter-exclude-job'),
+    reXTest: read('filter-exclude-test'),
```
Comment (liggitt) [inline triage/interactive.js]: these aren't part of the url params are they? if so, would be good to keep those stable

**Comment by liggitt** [comment]
I like this. Just had one question.

File: triage/interactive.js [hunk 2]
```diff
@@ -118,9 +121,12 @@ function setOptionsFromURL() {
   write('date', qs.date);
   write('job-ci', qs.ci);
   write('job-pr', qs.pr);
-  write('filter-text', qs.text);
-  write('filter-job', qs.job);
-  write('filter-test', qs.test);
+  write('filter-include-text', qs.text);
```
Comment (spiffxp) [inline triage/interactive.js]: @liggitt url params are filled in here, they remain unchanged

File: triage/script_test.js [hunk 3]
```diff
@@ -37,17 +37,17 @@ describe('Clusters', () => {
...
         ]};
         let first = {text: 'afirst', key: 'afirst', id: 'def0', tests: [
             {name: 'something', jobs: [{name: 'firstjob', builds: [5, 6]}]},
         ]};
-        expect('filters by text', [ham], [ham, spam], {reText: /ham/im, ci: true});
-        expect('filters by test', [ham], [ham, spam], {reTest: /volume/im, ci: true});
-        expect('filters by job', [ham], [ham, spam], {reJob: /cure/im, ci: true});
+        expect('filters by text', [ham, spam], [ham, spam, pr], {reText: /am/im, reXText: /b/im, ci: true});
+        expect('filters by test', [spam], [ham, spam, first], {reTest: /ing/im, reXTest: /some/im, ci: true});
+        expect('filters by job', [ham], [ham, spam], {reJob: /cure/im, reXJob: /g/im, ci: true});
```
Comment (BenTheElder) [inline triage/script_test.js]: needs more green eggs

**Comment by BenTheElder** [comment]
/lgtm
dr. seuss aside

**Pull request merged**